renderdoc/driver/gl/gl_debug.cpp

```diff
@@ -2249,8 +2249,8 @@ ResourceId GLReplay::RenderOverlay(ResourceId texid, CompType typeHint, DebugOve
     }
     else
     {
-      gl.glTextureImage2DEXT(DebugData.overlayTex, texBindingEnum, 0, eGL_RGBA16, texDetails.width,
-                             texDetails.height, 0, eGL_RGBA, eGL_FLOAT, NULL);
+      gl.glTextureImage2DEXT(DebugData.overlayTex, texBindingEnum, 0, eGL_RGBA16I, texDetails.width,
+                             texDetails.height, 0, eGL_RGBA_INTEGER, eGL_SHORT, NULL);
```

Comment:
According to https://www.khronos.org/registry/OpenGL-Refpages/es3.0/html/glTexImage2D.xhtml
neither GL_RGBA16F and GL_RGBA32F are color renderable formats, and as a result I was getting a debug message from WrappedOpenGL::DebugSnoop.

Having said that, it does seem strange that OpenGL does not provide support for color renderable float rendertargets. So after some digging I realized that this requires the EXT_color_buffer_float extension.

I'll revert this change and see if there's a way to do an extension check somewhere.